### DISCUSSION TITLE
Account or Settings Data

### DISCUSSION THREAD

**Discussion by Shomnipotence** [DiskTools/EdgeEx#32] [General]
account or settings data

Are new local accounts allowed? Or is it possible to access a Google account? or some other way?

**Comment by kitUIN** [comment]
If I write, then only local accounts , if someone else writes, then all in. XD

**Comment by jarno9981** [comment]
> account or settings data
> 
> Are new local accounts allowed? Or is it possible to access a Google account? or some other way?

You mean google account login so you can use youtube premium and gmail stuff
